packages/valory/skills/safe_deployment_abci/rounds.py

```diff
@@ -64,58 +72,58 @@ class SynchronizedData(BaseSynchronizedData):
     @property
     def safe_contract_address(self) -> str:
         """Get the safe contract address."""
-        return cast(str, self.db.get_strict("safe_contract_address"))
+        return cast(str, self.db.get_strict(DBKeys.SAFE_CONTRACT_ADDRESS.value))
 
 
 class RandomnessSafeRound(CollectSameUntilThresholdRound):
     """A round for generating randomness"""
 
-    round_id = "randomness_safe"
+    round_id = States.RANDOMNESS_SAFE.value
     allowed_tx_type = RandomnessPayload.transaction_type
-    payload_attribute = "randomness"
+    payload_attribute = RandomnessPayload.randomness.fget.__name__  # type: ignore
```

Comment:
It's annoying that we have to add a type ignore as the whole point was to have the static check. But mypy has a bug: https://github.com/python/mypy/issues/6185

In any case, once we refactor this as part of payload refactor this can be improved upon.

At least now the link is explicit.